contracts/GateSeal.vy

```diff
@@ -86,33 +91,46 @@ SEALING_COMMITTEE: immutable(address)
 # Sealing can be partial, meaning the committee may decide to pause only a subset of this list,
 # though GateSeal will still expire immediately.
 
-# The duration of the seal in seconds. This period cannot exceed 14 days. 
+# The duration of the seal in seconds. This period cannot exceed 21 days.
 # The DAO may decide to resume the contracts prematurely via the DAO voting process.
 SEAL_DURATION_SECONDS: immutable(uint256)
 
+DAO: immutable(address)
+EXTENSION_DURATION_SECONDS: immutable(uint256)
```

Comment:
Wouldn't it make more sense to decide on the extension duration at runtime? There may be situations (I imagine) where you don't want to prolong gate seal by too much.